packages/desktop/src/renderer/index.tsx

```diff
@@ -73,16 +73,22 @@ const store = createReduxStore({
 store.subscribe(() => {
     const state = store.getState();
     const csvColumns = interaction.selectors.getCsvColumns(state);
+    const displayAnnotations = selection.selectors.getAnnotationsToDisplay(state);
     const userSelectedApplications = interaction.selectors.getUserSelectedApplications(state);
     const hasUsedApplicationBefore = interaction.selectors.hasUsedApplicationBefore(state);
     const appState = {
         [PersistedConfigKeys.CsvColumns]: csvColumns,
+        [PersistedConfigKeys.DisplayAnnotations]: displayAnnotations.map((annotation) => ({
+            annotationDisplayName: annotation.displayName,
+            annotationName: annotation.name,
+            description: annotation.description,
+            type: annotation.type,
+        })),
         [PersistedConfigKeys.UserSelectedApplications]: userSelectedApplications,
     };
     if (JSON.stringify(appState) !== JSON.stringify(persistentConfigService.getAll())) {
         persistentConfigService.persist({
-            [PersistedConfigKeys.CsvColumns]: csvColumns,
-            [PersistedConfigKeys.UserSelectedApplications]: userSelectedApplications,
+            ...appState,
             [PersistedConfigKeys.HasUsedApplicationBefore]: hasUsedApplicationBefore,
```

Comment:
Seems like this should also be in the `appState` object - this isn't because of your changeset just seems buggy because otherwise isn't this always true?
`JSON.stringify(appState) !== JSON.stringify(persistentConfigService.getAll())`
If you agree I'd put `[PersistedConfigKeys.HasUsedApplicationBefore]: true` into the `appState` object.

Reply:
resolved in [f3b21ab](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/75/commits/f3b21abbf790b9287163be67ba25e771407927e0)